rotkehlchen/chain/ethereum/modules/uniswap/utils.py

```diff
@@ -32,6 +44,14 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
+UNISWAP_NFT_MANAGER_ADDRESS = string_to_ethereum_address(to_checksum_address('0xc36442b4a4522e871399cd717abdd847ab11fe88'))  # noqa: 501
+POOL_INIT_CODE_HASH = '0xe34f199b19b2b4f47f68442619d555527d244f78a3297ea89325f843f87b8b54'
+UNISWAP_V3_POSITIONS_PER_CHUNK = 10
+UNISWAP_V3_FACTORY_ADDRESS = string_to_ethereum_address('0x1F98431c8aD98523631AE4a59f267346ea31F984')  # noqa: 501
+UNISWAP_V3_DEPLOYED_BLOCK = 12369651
+UNISWAP_V3_ERROR_MSG = 'Remote error calling multicall contract for uniswap v3 {} for address properties: {}'  # noqa: 501
+SQRTX96 = 2**96
```

Comment:
This is not the square root. I would say `POW_96` is a better name... or other that you can come up with.